lang/semgrep-grammars/src/semgrep-julia/grammar.js

```diff
@@ -27,6 +27,40 @@ module.exports = grammar(base_grammar, {
       optional($._block),
     )),
 
+    // Metavariables
+    // We allow an identifier to be a simple metavariable regex, so that
+    // we properly support metavariables.
+    // Low precedence, so that we take less priority than parsing as
+    // an interpolation expression, if possible. See the comments below.
+    identifier: ($, previous) =>
+      prec(1, choice(
+        previous,
+        /\$[A-Z][a-zA-Z0-9]*/,
```

Comment:
we usually factorize the regexp in a separate rule, like _semgrep_metavariable: /$ ... /